src/ngx_http_lua_sleep.c

```diff
@@ -159,11 +159,18 @@ ngx_http_lua_sleep_cleanup(void *data)
     }
 
 #ifdef HAVE_POSTED_DELAYED_EVENTS_PATCH
+    ngx_event_t     *ev;
```

Comment:
We should never declare variables in the middle of a code block. This will lead to compilation errors in some C compilers.

Reply:
@agentzh Has been removed.